scripts/measures/build-2019-measures

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env bash
-
 currentPerformanceYear=$1
```

Comment:
you can set this to =2019 now, no need to pass it in

also for consistency you can have this file be measures/2019/build-measures